howto.md

```diff
@@ -1,14 +1,31 @@
 ---
-title: How to use them
+title: How to Run a Delta Chat Bot
 ---
 
-# How to use
+# How to Run a Delta Chat Bot
+
+To run a bot, you don't need to be a programmer. 
+You don't even need to own or rent a server.
+A bot runs on any computer.
+
+Any bot has a documentation about how to use it - you need to find this information.
+🚧 **TODO**: Where to find the bot's documentation?
+
+## Internet Connection
 
 For running any Delta Chat bot, there is one basic requirement: That a network connection to your email provider is possible via IMAP and SMTP.
 
-Next, checkout the documentation of the bot of your choice. You probably need an installation of the runtime that the bot uses (e.g. Python or NodeJS), but you might already have that.
+## Install the Bot
+
+Next, checkout the documentation of the bot of your choice.
+You need an installation of the runtime (programming language) that the bot uses (e.g. Python or NodeJS), if you don't have that already.
```

Comment:
Why not just write:
> To install a bot, follow the installation step in its `Readme.md` file.

Reply:
Because the READMEs don't really explain how to install the runtime - and people who are not developers have regularly problems with installing NodeJS for example, as it's really hard to know what you should even search for. This page is supposed to give a complete overview on what to do and helpful extra information.

But I also added that information above :)